content/sql/shop-project-part-2/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+content_type: project
+flavours:
+- none
+prerequisites:
+  hard:
+  - sql/shop-project
+  soft: []
+story_points: 3
+from_repo: sql/shop-project
+submission_type: continue_repo
+tags:
+- sql
+- skill/databases
+ready: false
+title: Shop Database using sql - part 2
+---
+
+In the last part of this project you created a Shop Database and performed some SQL queries, now it is time to add more queries.
+
+## Instructions
+
+Create a new branch and ensure your repository maintains the same structure with additional new queries.
+
+### Querying a database
+
+Save all of your instructions in script files.
+
+**NB! Be sure to label your answers by placing the question above the corresponding code. Submitting a wall of code without explanations makes it difficult to review.**
+
+18. Using `DISTINCT`, select all unique cities from the `Customers` table.
+19. Select all unique countries from `Customers` table.
+20. Select all unique job titles from the `Employees` table. 
+21. Classify `Customers` by `Country` into "Local" or "International".
+22. Classify `Orders` into "Completed" or "Pending" based on their status.
+23. Select Customer Full Names and Emails with Aliases.
+24. Select Products with their names and Price Aliased.
+25. With the use of an `INNER JOIN`, create a query that will join `Payments` with `Customers` full names using Aliases.
+26. With the use of an `INNER JOIN`, create a query that will join `Orders` with `Employees` full names using Aliases.
```

Comment:
Would it not be better to mention what the aliases should be named in each of these questions?